zerver/lib/bugdown/api_arguments_table_generator.py

```diff
@@ -105,7 +105,7 @@ def render_table(self, arguments: List[Dict[str, Any]]) -> List[str]:
         tr = """
 <tr>
   <td><code>{argument}</code></td>
-  <td><code>{example}</code></td>
+  <td class="example"><code>{example}</code></td>
```

Comment:
Probably not a big deal, but could you please rename this class to `json-api-example` to be more specific? Thanks!